cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/CaReconciler.java

```diff
@@ -375,38 +375,29 @@ Future<Void> reconcileClusterOperatorSecret(Clock clock) {
 
     /**
      * Perform a rolling update of the cluster so that CA certificates get added to their truststores, or expired CA
-     * certificates get removed from their truststores. Note this is only necessary when the CA certificate has changed
-     * due to a new CA key. It is not necessary when the CA certificate is replace while retaining the existing key.
+     * certificates get removed from their truststores. Note this is only necessary when the Cluster CA certificate has changed
```

Comment:
I'm trying to understand this doc and the method name. The doc says it's only needed for the cluster CA, but the method name doesn't make that distinction. Perhaps it should?

Reply:
Thanks @tombentley that's a good point. I've updated the method name to reflect the fact it's rolling the cluster CA, but also to reflect the fact that it doesn't always roll. I've also extended the Javadoc so I can remove the comment lower down.